lib/plugins/aws/package/compile/events/s3/index.js

```diff
@@ -21,7 +45,10 @@ class AwsCompileS3Events {
 
   newS3Buckets() {
     const bucketsLambdaConfigurations = {};
+    const bucketsLogicalIds = {};
```

Comment:
It'll be nice to use `Map` here. Note that such plain object notation is not safe for hacky bucket names as `hasOwnProperty`